extensions/wikia/Search/WikiaSearchController.class.php

```diff
@@ -244,9 +184,9 @@ public function searchVideosByTitle() {
 			throw new Exception( "Please include a value for 'title'." );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS112)